spec/lib/account_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+
+describe BankAccounts::Account do
+
+  describe "class methods" do
+    it "responds to 'new'" do
```

Comment:
Technically, you shouldn't need this test as all classes inherit from Object, which has the new method. What you could check is whether or not calling `BankAccounts::Account.new(x,y)` creates an Account object will the right attributes.